angular/src/app/modules/admin/cvstatus/create-update-cvstatus/create-update-cvstatus.component.spec.ts

```
@@ -0,0 +1,25 @@
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ CreateUpdateCvstatusComponent ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(CreateUpdateCvstatusComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
```

Biome

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)